### PR TITLE
Refresh: update Edge doc with Digital Ocean managed K8s example

### PR DESCRIPTION
**Pull request by matthijskrul**
## Summary of the changes / Why this is an improvement
- Provides Digital Ocean example for how to run CrateDB Edge with managed K8s
- Various wording fixes for consistency

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/edge.rst [hunk 1]
```diff
@@ -326,16 +331,115 @@ into the form.
...
+Install CrateDB Edge using third-party tools
+============================================
+
+You can combine CrateDB Edge with third-party tools for ease of use, such as
+managed Kubernetes providers and preconfigured Kubernetes distributions. In the
+former category, we have tested AWS, Azure, Digital Ocean, and GCP with the
+CrateDB Edge stack, and for the latter the distributions described below. In
+the walkthroughs below, we provide by way of example a guide for using
+`Digital Ocean`_ as a managed Kubernetes provider with CrateDB Edge, and guides
+for two of the most common Kubernetes distributions: `Microk8s`_ and `K3s`_.
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
for two of the most common Kubernetes distributions: `microk8s`_ and `k3s`_.
```

File: docs/edge.rst [hunk 2]
```diff
@@ -326,16 +331,115 @@ into the form.
 
 .. _edge-tools:
 
-Install CrateDB Edge using an on-premise Kubernetes distribution
-================================================================
+Install CrateDB Edge using third-party tools
+============================================
+
+You can combine CrateDB Edge with third-party tools for ease of use, such as
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
You can combine CrateDB Edge with tools for ease of use, such as
```

File: docs/edge.rst [hunk 3]
```diff
@@ -326,16 +331,115 @@ into the form.
 
 .. _edge-tools:
 
-Install CrateDB Edge using an on-premise Kubernetes distribution
-================================================================
+Install CrateDB Edge using third-party tools
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
Install CrateDB Edge using managed k8s
```

File: docs/edge.rst [hunk 4]
```diff
@@ -326,16 +331,115 @@ into the form.
...
+============================================
+
+You can combine CrateDB Edge with third-party tools for ease of use, such as
+managed Kubernetes providers and preconfigured Kubernetes distributions. In the
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
managed Kubernetes providers and self-hosted Kubernetes setups. In the
```

File: docs/edge.rst [hunk 5]
```diff
@@ -326,16 +331,115 @@ into the form.
...
-we responsible for their behavior. That said, we have tested the instructions
-provided below for functionality. Users less familiar with customizing their
-Kubernetes stack on their own may find either of these two guides a practical
-solution for easier CrateDB Edge setup.
+
+Create Kubernetes cluster
+'''''''''''''''''''''''''
+
+Create a Kubernetes cluster using the Digital Ocean cloud interface, under
+"Manage", then "Kubernetes". When configuring the machine, make sure to choose
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
"Manage", then "Kubernetes". When configuring the cluster, make sure to choose
```

File: docs/edge.rst [hunk 6]
```diff
@@ -326,16 +331,115 @@ into the form.
...
+
+Eventually, the script will indicate that there is no ``crate-premium`` storage
+class available. To define this storage class correctly, copy the default
+storage class Digital Ocean provides, then change the the ``name`` to
+``crate-premium`` in the copied file. For example, using kubectl and Vim:
+
+.. code-block:: console
+
+    kubectl get sc do-block-storage -o yaml > sc.yaml
+    vi sc.yaml
```
Comment (WalBeh) [inline docs/edge.rst]:
```suggestion
  kubectl get sc do-block-storage -o yaml | grep -vi is-default-class | sed -e 's/name: do-block-storage/name: crate-premium/' | kubectl create -f -
```

**Review by WalBeh — APPROVED** [review]

File: docs/edge.rst (hunk 1 above)
Comment (matthijskrul) [reply docs/edge.rst]: They write it as "MicroK8s" and "K3s" themselves so I would follow that actually

**Review by RomanMeingassner — COMMENTED** [review]
Looks good to me

**Pull request merged**